Bio/Seq.py

```diff
@@ -929,21 +966,19 @@ def transcribe(self):
         >>> coding_dna.transcribe()
         Seq('AUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAG')
 
-        Trying to transcribe a protein or RNA sequence raises an exception:
+        Trying to transcribe an RNA sequence should have no effect.
```

Comment:
Here also I would raise a `ValueError` if a "U" is found in the sequence

Reply:
If designing this from scratch, I might agree - but changing this historic behaviour seems unnecessary. Someone doing a lot of work with RNA is bound to have tried redundant transcribe calls to ensure their nucleotide input is now RNA.

Reply:
If we did do this, we should add a similar check for T on the back-transcribe method.

Reply:
OK then let's just put a warning in the documentation.